rustfmt.toml

```diff
@@ -1,10 +1,5 @@
-max_width = 100
-reorder_imports = true
-reorder_modules = true
-
-# == Nightly only. ==
-# imports_indent = "Block"
-# imports_layout = "Mixed"
-# imports_granularity = "Crate"
-# group_imports = "Preserve"
-# reorder_impl_items = false
+imports_indent = "Block"
+imports_layout = "Mixed"
+imports_granularity = "Crate"
+group_imports = "Preserve"
+reorder_impl_items = false
```

Comment:
I recommend these from SPL:

```toml
comment_width = 80
edition = "2021"
group_imports = "One"
imports_granularity = "One"
wrap_comments = true
```

Reply:
Cool, we should update the template with a set of defaults - the ones that I am using are from the template – we could use the ones that you are recommending.